.ci/windows-steps.yaml

```
@@ -12,8 +12,8 @@ steps:
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_random-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install unofficial-flayan-cereal -o $(Agent.ToolsDirectory)
+    nuget install ensmallen -o $(Agent.ToolsDirectory) -Version 2.17.0
```

```suggestion
dir $(Agent.ToolsDirectory)
```
Maybe this will show us more?